tsp/xy_model.py

```diff
@@ -118,8 +118,9 @@ def __init__(self, params):
         # Array of tags for each XY value, used when a value passes a filter
         self.tags = []
         if TAGS_KEY in params:
-            for tag in params.get(TAGS_KEY):
-                self.tags.append(tag)
+            if params.get(TAGS_KEY) != None:
```

Comment:
The tags are not in the TSP spec nor are they serialized by the Trace Compass server, even though Trace Compass xy data providers might create them. Or do I miss something?

We should update the specification.